packages/apps/frontera/src/routes/organizations/src/components/Columns/Cells/socials/OrganizationLinkedInCell.tsx

```diff
@@ -82,13 +82,13 @@ export const OrganizationLinkedInCell = observer(
       );
     }
 
-    const linkedIn = organization.value.socials.find((social) =>
+    const linkedIn = organization.value.socialMedia.find((social) =>
       social.url.includes('linkedin'),
     );
     if (!linkedIn?.url) return null;
 
     const formattedLink = getFormattedLink(linkedIn.url).replace(
-      /^linkedin\.com\/in\//,
```

Comment:
this was the valid url for a valid linkedin profile otherwise the URL is wrong ( /in --> profile /company -->companies profile ) so we are sure we want to save some wrong url's ?